package.json

```diff
@@ -47,47 +47,42 @@
     "@commitlint/cli": "^8.2.0",
     "@commitlint/config-conventional": "^8.2.0",
     "@schemastore/package": "^0.0.5",
-    "@semantic-release/changelog": "^3.0.5",
-    "@semantic-release/git": "^7.0.17",
+    "@semantic-release/changelog": "^5.0.1",
+    "@semantic-release/git": "^9.0.0",
     "@types/eslint": "^6.1.3",
     "@types/jest": "^25.1.0",
-    "@types/node": "^12.6.6",
+    "@types/node": "^13.13.5",
     "@typescript-eslint/eslint-plugin": "^2.5.0",
     "@typescript-eslint/parser": "^2.5.0",
-    "babel-jest": "^25.2.0",
+    "babel-jest": "^26.0.1",
     "babel-plugin-replace-ts-export-assignment": "^0.0.2",
-    "eslint": "^5.1.0 || ^6.0.0",
+    "eslint": "^5.1.0 || ^6.0.0 || ^7.0.0",
     "eslint-config-prettier": "^6.5.0",
     "eslint-plugin-eslint-comments": "^3.1.2",
     "eslint-plugin-eslint-plugin": "^2.0.0",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-node": "^11.0.0",
     "eslint-plugin-prettier": "^3.0.0",
-    "husky": "^3.0.9",
-    "jest": "^25.2.0",
-    "jest-runner-eslint": "^0.7.1",
-    "lint-staged": "^9.4.2",
-    "prettier": "^1.19.1",
-    "prettylint": "^1.0.0",
+    "husky": "^4.2.5",
+    "jest": "^26.0.1",
+    "jest-runner-eslint": "^0.8.0",
+    "lint-staged": "^10.2.2",
+    "prettier": "^2.0.5",
     "resolve-from": "^5.0.0",
     "rimraf": "^3.0.0",
-    "semantic-release": "^15.13.28",
+    "semantic-release": "^17.0.7",
     "typescript": "^3.5.3"
```

Comment:
nit: Would be cool to bump typescript - while symbolic, I feel it's nice to have it at a higher number.

Reply:
it's bumped to latest in the lockfile